index.bs

```diff
@@ -325,6 +325,16 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     that this does not constitute [=user verification=] because [=TUP=], by definition, is not capable of [=biometric
     recognition=], nor does it involve the presentation of a shared secret such as a password or PIN.
 
+: <dfn>Client-side-resident Credential Private Key</dfn>
+:: A [=Client-side-resident Credential Private Key=] is stored either on the client platform, or in some cases on the authenticator
```

Comment:
This is where it gets tricky.  The Authenticator always controls/protects/owns the key, but it might offload the persistent storage of the wrapped key to some external component (if this is what is meant by resident).
We need a description that makes this distinction clear. 
My proposal:
By definition, the private key is always exclusively controlled by the Authenticator. In the case of a [=Client-side-resident Credential Private Key=], the Authenticator might offload storage of wrapped key material to the client platform, but the client platform is not expected to offload the key storage to remote entities (e.g. RP Server).